mockup/patterns/relateditems/pattern.js

```diff
@@ -99,13 +99,12 @@ define([
       homeText: _t('home'),
       folderTypes: ['Folder'],
       selectableTypes: null, // null means everything is selectable, otherwise a list of strings to match types that are selectable
-      attributes: ['UID', 'Title', 'portal_type', 'path', 'getIcon'],
+      attributes: ['UID', 'Title', 'portal_type', 'path'],
       dropdownCssClass: 'pattern-relateditems-dropdown',
       maximumSelectionSize: -1,
       resultTemplate: '' +
         '<div class="pattern-relateditems-result pattern-relateditems-type-<%= portal_type %> <% if (selected) { %>pattern-relateditems-active<% } %>">' +
         '  <a href="#" class="pattern-relateditems-result-select <% if (selectable) { %>selectable<% } %> contenttype-<%= portal_type.toLowerCase() %>">' +
-        '    <% if (typeof getIcon !== "undefined" && getIcon) { %><span class="pattern-relateditems-result-icon"><img src="<%= getIcon %>" /></span><% } %>' +
```

Comment:
@vangheem 
I am afraid, I don't yets see clear through all this code.

What i tried is 
...
```
<span class="pattern-relateditems-result-title contenttype-<%= portal_type.toLowerCase() %>"><%= Title %></span>'
```
which is a hack: it should work for 'Folder', 'Image' but of course not for 'News Item' or other (custom ...).
(need something like normalizeString here, or add contenttype-class to vocabulary?

but also classes like 'contenttype-folder' are not respected (need to fix in barcelonetta less?)

using thumb here would be a good and easy solution.  But:
* images (64x64) as i implemented it for folder contents might be to big for the dropdown
* we should prevent rendering class=contenttype-something together with thumb ..
  so something like: 
``` `
if not thumb:  class=contenttype.something 
                                 else <img src = geturl()/@@images/image/icon> 
```